python/ray/dataframe/dataframe.py

```diff
@@ -23,10 +23,10 @@ def __init__(self, df, columns):
         self.columns = columns
 
     def __str__(self):
-        return str(pd.concat(ray.get(self.df)))
+        return "ray.DataFrame object"
```

Comment:
Good call, it may make sense to just use the default implementation here (same with `__repr__`).

Making `__str__` and `__repr__` blocking calls makes them a bit hard to use in the interpreter :)